templates/SilverStripe/Admin/Includes/LeftAndMain_MenuList.ss

```diff
@@ -2,10 +2,10 @@
 	<% loop $MainMenu %>
 		<li class="$LinkingMode $FirstLast <% if $LinkingMode == 'link' %><% else %>opened<% end_if %>" id="Menu-$Code" title="$Title.ATT">
 			<a href="$Link" $AttributesHTML>
-				<% if IconClass %>
+				<% if $IconClass %>
 					<span class="menu__icon $IconClass"></span>
 				<% else %>
-					<span class="menu__icon font-icon-circle-star font-icon-{$Icon}">&nbsp;</span>
+					<span class="menu__icon icon icon-16 icon-{$Icon}">&nbsp;</span>
```

Comment:
This means that if no custom icon is set for a new `LeftAndMain` section, it’s blank instead of the fallback star icon. I have an idea I’ll post below...

Reply:
Thanks Loz. I have added your suggested change.